providers/hns/hns_roce_u.h

```diff
@@ -206,6 +209,13 @@ struct hns_roce_wq {
 	int				offset;
 };
 
+/* record the result of sge process */
+struct hns_roce_sge_info {
+	unsigned int		valid_num; /* sge length is not 0 */
```

Comment:
Please don't do vertical space alignment in newly added code.

Thanks

Reply:
OK, I have modified them and pushed a new version. Thank you.